src/transformStateWithClones.js

```diff
@@ -8,6 +8,42 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = { ...state };
+  const arrResult = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i]['type'] === 'addProperties') {
```

Comment:
Use switch/case instead of `if`

Reply:
is not fixed. Please apply recommendations